index.js

```diff
@@ -662,16 +662,19 @@ class GithubScm extends Scm {
     async _getCommitRefSha(config) {
         try {
             const commit = await this.breaker.runCommand({
-                action: 'getCommitRefSha',
+                action: 'getCommit',
                 token: config.token,
                 params: {
                     owner: config.owner,
                     repo: config.repo,
-                    ref: config.ref
+                    ref: config.ref,
+                    mediaType: {
```

Comment:
Do we need additional `mediaType` info.? I can obtain `sha` without `mediaType`.

```
$ cat snipet.js
const Octokit = require('@octokit/rest');
const octokit = new Octokit({
  auth: '<masked>'
});

octokit.repos.getCommit({
  owner: 'tk3fftk',
  repo: 'sdctl',
  ref: 'v0.0.1'
}).then(r => console.log(r))
.catch(err => console.error(err));

$ node snipet.js
{ status: 200,
  url: 'https://api.github.com/repos/tk3fftk/sdctl/commits/v0.0.1',
  headers:
   { 'access-control-allow-origin': '*',
     'access-control-expose-headers':
      'ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type',
     'cache-control': 'private, max-age=60, s-maxage=60',
     connection: 'close',
     'content-encoding': 'gzip',
     'content-security-policy': 'default-src \'none\'',
     'content-type': 'application/json; charset=utf-8',
     date: 'Wed, 21 Aug 2019 10:38:01 GMT',
     etag: 'W/"a70af1bca9e2ed438e596bb96fc84939"',
     'last-modified': 'Sun, 21 Jul 2019 10:30:02 GMT',
     'referrer-policy': 'origin-when-cross-origin, strict-origin-when-cross-origin',
     server: 'GitHub.com',
     status: '200 OK',
     'strict-transport-security': 'max-age=31536000; includeSubdomains; preload',
     'transfer-encoding': 'chunked',
     vary:
      'Accept, Authorization, Cookie, X-GitHub-OTP, Accept-Encoding',
     'x-accepted-oauth-scopes': '',
     'x-content-type-options': 'nosniff',
     'x-frame-options': 'deny',
     'x-github-media-type': 'github.v3; format=json',
     'x-github-request-id': 'C259:780C:313344:435B07:5D5D1F09',
     'x-oauth-scopes': 'repo',
     'x-ratelimit-limit': '5000',
     'x-ratelimit-remaining': '4995',
     'x-ratelimit-reset': '1566387481',
     'x-xss-protection': '1; mode=block' },
  data:
   { sha: '7b71919649a33aedb9da889d87915c0e2cd13f0a',
(snip)
```

This `sha` is looks valid.
https://github.com/tk3fftk/sdctl/blob/7b71919649a33aedb9da889d87915c0e2cd13f0a/README.md

Reply:
Thank you for teaching.
Is better simple and easy to understand of not use "mediaType".

Reply:
this fixed